Add RM option for enabling for deployment

rovinbhandari opened the pull request:
Since Azure RM supports enabling a keyvault for deployment, it should be mentioned as an option, especially when other operations being done in this context use Azure RM.

azurecla commented:
Hi __@joseidz__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (joseidz). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

rovinbhandari committed:
Add RM option for enabling for deployment

Since Azure RM supports enabling a keyvault for deployment, it should be mentioned as an option, especially when other operations being done in this context use Azure RM.
